travis/licenses_golden/licenses_flutter

```diff
@@ -202,6 +202,11 @@ FILE: ../../../flutter/lib/ui/window/window.h
 FILE: ../../../flutter/shell/common/skia_event_tracer_impl.cc
 FILE: ../../../flutter/shell/platform/android/apk_asset_provider.cc
 FILE: ../../../flutter/shell/platform/android/io/flutter/plugin/common/JSONUtil.java
+FILE: ../../../flutter/shell/platform/android/io/flutter/plugin/platform/PlatformView.java
```

Comment:
These paths should be moved to a different position in the license file (check the Travis logs)

Reply:
regenerated the licenses files... not sure how did this happen